app/components/account/MetaplexNFTHeader.tsx

```diff
@@ -53,13 +60,13 @@ export function MetaplexNFTHeader({ nftData, address }: { nftData: NFTData; addr
                 {<h6 className="header-pretitle ms-1">Metaplex NFT</h6>}
                 <div className="d-flex align-items-center">
                     <h2 className="header-title ms-1 align-items-center no-overflow-with-ellipsis">
-                        {metadata.data.name !== '' ? metadata.data.name : 'No NFT name was found'}
+                        {metadata.name !== '' ? metadata.name : 'No NFT name was found'}
```

Comment:
I'm not opposed to changing where this sources name from, but do you mind explaining rationale for this change & others that use `metadata` instead of `metadata.data`?